recipes/sfml/all/conanfile.py

```diff
@@ -105,8 +105,6 @@ def package(self):
         cmake.install()
 
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-        os.remove(os.path.join(self.package_folder, "license.md"))
-        os.remove(os.path.join(self.package_folder, "readme.md"))
```

Comment:
These files are installed by sfml and must be removed in the CCI recipe. Without these lines, the hooks raise the KB-H013 error 
```
[HOOK - conan-center.py] post_package(): ERROR: [DEFAULT PACKAGE LAYOUT (KB-H013)] Unknown file 'license.md' in the package (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H013)
[HOOK - conan-center.py] post_package(): ERROR: [DEFAULT PACKAGE LAYOUT (KB-H013)] Unknown file 'readme.md' in the package (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H013)
```

Reply:
On what platform? Those files do not exist on Linux and packaging fails with a hard error as stated earlier. Rather, they get installed in share/SFML/. Looking into the CMakeLists.txt for SFML it looks like it varies based on platform. 💯💯

I'll make the changes.

Reply:
I figured the simplest way to handle it is to patch the install commands out of the CMakeLists.txt file since we're patching out other things anyway.